js/mage/adminhtml/product.js

```diff
@@ -1001,14 +1001,18 @@ function toogleFieldEditMode(toogleIdentifier, fieldContainer) {
 }
 
 function disableFieldEditMode(fieldContainer) {
-    $(fieldContainer).disabled = true;
+    if ($(fieldContainer)) {
```

Comment:
from my tests, this check is not necessary, but it doesn't seem bad to have it anyway

Reply:
the problem is that `Varien_Data_Form_Element_Label` doesn't do a HTML Element with the Id of the attribute around the label text.

so then Prototype tries to access `$(fieldContainer)` it is null